### ISSUE TITLE
A url exists in sentence - o

### ISSUE THREAD

**Issue by AlizKaz** [Text-Mining/Persian-NER#54]
### Description

The sentence seems to have invalid contents. 

![invalid_sentence](https://user-images.githubusercontent.com/44096740/51429022-8e641b80-1c1f-11e9-99da-1015d9d7dafe.png)

### Steps to Reproduce

This issue cannot be reproduced.

**Expected behavior:** 

It is expected to see a word in sentence.

**Actual behavior:**

The first part of a sentence is a url which is not expected.

### Versions

```
Browser: (chrome Version 71.0.3578.98 (Official Build) (64-bit))
OS: (windows 7)
Platform: (web)
```

**Comment by Hameds** [comment]
This is a dataset related issue. however, we change backend API and we will try to fix avoid this kind of words in UI

**Comment by Hameds** [comment]
@AlizKaz no need to close it for now

**Comment by Hameds** [comment]
we made a change in API so from now, URLs like this will be filtered from output

**Comment by Hameds** [comment]
Reopen beacuse of another occurance

![capture](https://user-images.githubusercontent.com/1385656/52909781-2f87d580-32a3-11e9-993a-1759e1ac5f7c.PNG)

**Comment by Hameds** [comment]
fixed (`v2.11.2.0`)
